benefits/core/models.py

```diff
@@ -282,9 +282,12 @@ class EnrollmentFlow(models.Model):
         blank=True,
         help_text="A space-separated list of identifiers used to specify what access privileges are being requested",
     )
-    claims_claim = models.TextField(
+    claims_eligibility_claim = models.TextField(
         null=True, blank=True, help_text="The name of the claim (name/value pair) that is used to verify eligibility"
     )
+    claims_extra_claims = models.TextField(
+        null=True, blank=True, help_text="A space-separated list of any additional claims (name/value pair)"
```

Comment:
I think this help text is a little misleading (I see you just grabbed it from the item above).

This isn't a name/value pair right? It's just the name of the claim. Can we edit these help texts?

Reply:
Yep, I'm sure I was thinking about scopes and thought that name/value made sense but these are claims, thanks for pointing this out.

I implemented the change in 1de70c6da2ef2cfaedd3e20cc68aed71d4201dd1 and to avoid data migration operations, used two steps in the Django migration, a `RenameField` and an `AlterField`. Otherwise, Django thinks `claims_eligibility_claim` is a brand new field and that would require migrating data.